premailer/premailer.py

```diff
@@ -510,6 +514,24 @@ def _load_external(self, url):
 
         return css_body
 
+    @staticmethod
+    def six_colour(colourValue):
+        '''Fix background colours for Lotus Notes
+
+        Notes which fails to handle three character ``bgcolor`` codes well.
+        see <https://github.com/peterbe/premailer/issues/114>'''
+
+        shortCodes = re.compile(r'^#([0-9A-F])([0-9A-F])([0-9A-F])$', re.I)
```

Comment:
If you use `re.I` you don't need to type `A-F`. The default thing is to write in lowercase.